donkey_gym/envs/donkey_sim.py

```diff
@@ -315,6 +315,22 @@ def send_control(self, steer, throttle):
         msg = {'msg_type': 'control', 'steering': steer.__str__(), 'throttle': throttle.__str__(), 'brake': '0.0'}
         self.queue_message(msg)
 
+    def send_regen_road(self, road_style=0, rand_seed=0, turn_increment=0.0):
+        '''

```

Comment:
Please use double quotes and follow the docstring style ;) (`:param road_style: (int) ...`)

Reply:
Will rework this PR according to https://github.com/hill-a/stable-baselines